### ISSUE TITLE
Sovol 40W Laser wiring

### ISSUE THREAD

**Issue by coldstorm82** [makerbase-mks/MKS-DLC32#303]
Hi Gentlemen,

I own the Sovol 40W diode laser (https://sovol3d.com/products/40-w-laser-engraver-kits-for-3d-printer-cnc-machine-laser-engraving-cutting-5w-output), but I have problems with wiring.

The laser has two two-pin connectors at the top, one marked with Power (+ -) the other with PWM (+ -). I couldn't figure out how to connect it to the MKS DLC32 V2. 

Currently I attached Power to the spindle-connector and I'm able to use the laser with different power settings.

But how do I connect PWM? To the TTL-pin? Or do I have to connect PWM to the spindle and take Power from another connector? I don't want to fry my laser ;-).

Thanks for your support!

**Comment by manuelchaves** [comment]
Ok, on the board, upper right corner, you have a S-TTL-V connector where we have Signal +12/24V and -.Connect the - and the TTL to the respective connector of your SOVOL 40W, resolved the TTL signal. As for power, it is recommended to connect this S-TTL-V (+ 12/24V and -) to a MOS switching module, like this one, and using direct power from the power supply and never powered directly by the MKS DLC32. Was I able to help you?
![Makerbase-mks-dlc32-offline-cont](https://user-images.githubusercontent.com/8344682/207287042-fc713506-f530-4818-abfc-ca627a346fb2.jpg)
![costas](https://user-images.githubusercontent.com/8344682/207287063-86940a79-2c3f-4810-af77-3990f23edd14.jpg)
![M-dulo-de-acionamento-pwm-do-int](https://user-images.githubusercontent.com/8344682/207287097-406ecb0c-66a7-4f54-8c79-af30f8ebff5a.jpg)

**Comment by coldstorm82** [comment]
Hi Manuel,

thanks a lot for your help!!!

Just to make sure I understood you well, I did a little sketch for the wiring. Cause I'm not quite sure about the connection of the MOS module. Do I also need to connect the PWM signal to the MOS?

(I studied mechanical engineering, I'm not so deep in the electrical stuff 😅)

![image](https://user-images.githubusercontent.com/79671876/207298453-a0e4410e-7cbe-452d-9f2c-f0bd91ece9d7.png)

Again, thanks a lot!
